community/howto/samwiseg0/btt_smart_filament_sensor.md

```diff
@@ -21,7 +21,7 @@ The BTT Smart Filament Sensor has other advantages beyond letting you seamlessly
 1. Ensure the sensor is plugged into an endstop port. Please refer to your MCU pinout to identify the pin you are using. 
 1. Set your endstop pin as a pull-up (put a `^` before the pin).
 1. Ensure that the reverse bowden is contiguous all the way from the toolhead to the sensor. Any open/unconstrained filament will cause it to false detect
-1. Sensorless homing/StallGuard is known to interfere with the sensor if the diag pin is set on the MCU where the filament sensor is connected.
+1. When choosing pins on the MCU board to which you will connect the sensor, please be aware that the endstop pins are oftentimes connected with diag pins (for the given stepper), therefore using the seemingly available endstop pins might/will interfere with sensorless homing/StallGuard possibly set up for the stepper.
```

Comment:
Did something change? As far as I am aware dig pins are not set/enabled by default in any MCU today? The way this reads to me is that this is a default which it is not. 

I do agree that we can possibly change this statement but the user has to make a conscious effort to use sensorless homing/stallguard. 

I purpose changing it to this instead.

```
Sensorless homing/StallGuard is known to interfere with the sensor if the diag pin is set on the MCU on the corresponding endstop pin. For example, If the diag pin is enabled for stepper driver position 3 then endstop position 3 will not allow the filament sensor to be detected properly. 
```